content/md/what-is/understand-data-quality.md

```diff
@@ -16,10 +16,15 @@ Because rich, coherent and consistent product data is important in order to deli
 This new capability helps you measure your data quality along two axes: **enrichment** and **consistency**.  
 Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.
 
+::: tips
+By disabling chosen `attribute groups` in Akeneo `settings` , you can rate your enrichment and consistency status and progress on the most relevant product data for your end-customer needs. Therefore, you will get insights on what’s truly important to improve your Product Experience.
```

Comment:
```suggestion
By disabling specific `attribute groups` in Akeneo `settings`, you can focus your enrichment and consistency status and progress on the most relevant product data for your end-customer needs. Therefore, you will get insights on what’s truly important to improve your Product Experience.
```

Reply:
⚠️ consistency in uppercases. Here enrichment and consistency are in lowercase but above, l.29, uppercases.